helm/prometheus-operator/templates/clusterrole.yaml

```diff
@@ -28,8 +24,11 @@ rules:
 - apiGroups:
   - monitoring.coreos.com
   resources:
+  - alertmanager
```

Comment:
Is this resource required?

Reply:
```kubectl get thirdpartyresource``` yields:
```
NAME                                    DESCRIPTION                           VERSION(S)
alertmanager.monitoring.coreos.com      Managed Alertmanager cluster          v1alpha1
prometheus.monitoring.coreos.com        Managed Prometheus server             v1alpha1
service-monitor.monitoring.coreos.com   Prometheus monitoring for a service   v1alpha1
```

Prometheus-operator below 0.12.0 creates the ```alertmanager```, ```prometheus``` and ```service-monitor``` thirdpartyresources in group ```monitoring.coreos.com```.

Not being well versed in rbac, I assumed the ClusterRole's responsibility was to describe the set of accesses on resources that prometheus-operator's ServiceAccount requires. Hence my additions to the clusterrole.yaml. please let me know if these need to be removed.